tests/test_encryption.py

```diff
@@ -0,0 +1,56 @@
+import base64
+import json
+import os
+import unittest
+
+from unittest.mock import patch
+
+
+class EncryptionTest(unittest.TestCase):
+
+    def setUp(self):
+        with open('data/artifacts.json') as artifacts:
+            self.test_artifacts = json.load(artifacts)
+        os.environ['CIS_ARN_MASTER_KEY'] = self.test_artifacts['dummy_arn']
+
+    @patch('cis.encryption.kms')
+    @patch('cis.encryption.os')
+    def test_encrypt(self, mock_os, mock_kms):
+        test_kms_data = {
+            'Plaintext': base64.b64decode(self.test_artifacts['Plaintext']),
+            'CiphertextBlob': base64.b64decode(self.test_artifacts['CiphertextBlob'])
+        }
+        test_iv = base64.b64decode(self.test_artifacts['IV'])
+
+        mock_kms.generate_data_key.return_value = test_kms_data
+        mock_os.urandom.return_value = test_iv
+
+        expected_result = {
+            'ciphertext': base64.b64decode(self.test_artifacts['expected_ciphertext']),
+            'ciphertext_key': test_kms_data['CiphertextBlob'],
+            'iv': test_iv,
+            'tag': base64.b64decode(self.test_artifacts['expected_tag'])
+        }
+
+        from cis.encryption import encrypt
+        self.assertEqual(expected_result, encrypt(b'foobar'))
+
+    @patch('cis.encryption.kms')
```

Comment:
No comment.  But this is a cool way to test this without recording API calls.  <3